### ISSUE TITLE
Some attributes are not described in the docs

### ISSUE THREAD

**Issue by e-koch** [e-koch/FilFinder#49]
Some properties in FilFinder2D are not described in the docs (i.e. `FilFinder2D.skeleton_longpath`).

**Comment by pippo1990** [comment]
Hi I am trying to make use of your nice tool (would say more than just a tool) , I am using basic exaples just to find FilFinder2D.skeleton_longpath, is there any way to get the lenght of it in  pixels ? I tried branch_lenghts but is just sizes of the various segment found and cannot use them because I dont know which were selected for theh skelethon_longpath , unless I need tu sum them all because these are just the ones selected to create the seleton ?

**Comment by pippo1990** [comment]
dodgy I can run print('\nfil.branch_lenghts() :  ' , fil.branch_lengths()) but   print('\nfil.lenghts() :  ' , fil.lenghts())    gives AttributeError: 'FilFinder2D' object has no attribute 'lenghts'   I am on fil_finder.__version__  :  1.7.2

**Comment by e-koch** [comment]
Just checking: was the `lenghts` a typo in the comment above?

**Comment by pippo1990** [comment]
yep sorry, my right hand is faster than the left one sorry, It's working now

**Comment by pippo1990** [comment]
for one of my test fil.lengths is in between the number of pixel and the geometric distance calculated on the skeleton_longpath by hand (pixel_to_pixel) i.e.  :   my measures length 1315.8813844421359  vs number of pixels  1128 while fil.lengths is fil.lenghts() :   [1298.91082169] pix  any reason about that , is it geometric lenght ? I am not able to interpret the filament.py part describing lenght: line 331 filament.py

**Comment by e-koch** [comment]
There can be a moderate discrepancy between the geometric length and the one filfinder finds from the longest path. FilFinder generalizes the identification of the intersections between branches along the longest path to allow them to be made up of several pixels. The distance it uses if then a median of the intersection pixels, added to the geometric distances along each branch.

There's also an option when running `FilFinder2D.find_widths` where 2 * width is added to the filament length to account for the shortening from the skeletonization process: https://fil-finder.readthedocs.io/en/latest/api/fil_finder.FilFinder2D.html#fil_finder.FilFinder2D.find_widths. If you are running that step, you can disable this with `add_width_to_length=False`

**Comment by pippo1990** [comment]
thanks for your nice and complete explanation
